### PR TITLE
Add syntax highlight

### PR DESCRIPTION
**Pull request by lhuriguen**
Added syntax highlighting for code blocks.

### CODE REVIEW

File: authentication_authorization/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ urlpatterns = patterns('',
 
 Then we need a template for the login page, so create a directory `blog/templates/registration` and a file inside named `login.html`:
 
-```
+```django
```
Comment (aniav) [inline authentication_authorization/README.md]: I don't think gitbook has syntax for Django really. It's HTML AFAIK here :)

Comment (lhuriguen) [inline authentication_authorization/README.md]: I used the GitBook editor and it worked there. Is there any way to make sure?

Comment (helenst) [inline authentication_authorization/README.md]: highlight.js does have django support... I also gave it a quick try in the tutorial with gitbook (I can't get extensions running as it's gitbook 1.5 still!) and it seemed to work :smile:

**Comment by aniav** [comment]
Ok, checked that in highlight.js source code, and it seems I just didn't check that enough earlier. Thank you! :sparkling_heart: :+1:

**Pull request merged**